src/utils/buildMultipartBody.spec.ts

```diff
@@ -0,0 +1,13 @@
+import FormData from 'form-data';
+
+import { buildMultipartBody } from './buildMultipartBody';
+
+describe('buildMultipartBody', () => {
+  it('should return a Form data object', () => {
+    const assetPath = './__fixtures__/test.png';
+    const key = 'value';
```

Comment:
Not quite - did you happen to read up on the documentation for the multipart body spec? The key is the key that the value (assetPath is the value) is associated with. 

```suggestion
    const key = 'key';
```